spec/features/wiki/crud_spec.rb

```diff
@@ -13,9 +13,9 @@
     click_button "Log in"
 
     # localhost/wikis/new  wikis#index
-    expect(page).to have_content("Hello test@email.com")
+    expect(page).to have_content("Hey test@email.com")
     expect(current_path).to eq(wikis_path)
-    click_link "New wiki"
+    click_link "Public wiki" || "Private wiki"
```

Comment:
The `|| "Private wiki"` won't do anything here to change the behaviour of line 18. The line behaves the same as not having it. This is what's happening:

```ruby
click_link "Public wiki"
```